tools/src/commands/WorkflowDispatch.ts

```diff
@@ -64,6 +64,11 @@ const CUSTOM_WORKFLOWS = {
       checkAll: 'check-all',
     },
   },
+  'dogfooding-clients': {
+    name: 'Dogfooding Clients',
+    baseWorkflowSlug: 'dogfooding-clients',
+    inputs: {},
+  },
```

Comment:
It's probably unnecessary as it doesn't add any inputs to the base workflow, which is automatically detected by our scripts. This map is intended for custom workflow versions with predefined inputs 😉